.github/workflows/run_tests.yml

```diff
@@ -33,6 +33,10 @@ jobs:
           pip install -e .[tests]
           pip install -e .[docs]
           pip install -e .[dev]
+      - name: Server Architecture
```

Comment:
Is there any need to know the CPU or was this for debugging?

If the later this could be removed.

Reply:
With @jonbarker68 , we were wondering if the CPU can affect the output of mbstoi, resulting in errors in tests. 
Perhaps, resulting in differences after 5 decimals places. 

I am trying to find out why the test result of hasqi and haaqi returns always the same value locally but, a different value from the sixth decimal place when running in github. 
So, I am taking the opportunity to print the CPU to see if I can see a pattern.

Reply:
Ah ok, its on-going debugging.

You can create "Draft Pull requests" for this sort of thing and they should still start the tests (use the downward pointing arrow next to "Create Pull Request" to change selection when first creating).

Reply:
:open_mouth: 
I didn't know the draft PR, 
thank you.